scripts/build.mjs

```diff
@@ -61,6 +61,8 @@ async function main() {
       {
         ...pkg,
         files: ["dist-*/**", "bin/**"],
+        main: "dist-bundle/index.js",
+        types: "dist-types/index.d.ts",
         exports: {
           ".": {
             types: "./dist-types/index.d.ts",
```

Comment:
did you try to replace with `exports: "./dist-bundle/index.js"` and just add the `types` key to the pkg root?

Reply:
It's not just ncc, typescript as well that is having some issues resolving packages.

I have tried that syntax and I had some issues.

Might just try again

Reply:
> As for comments, we can leave one in the build file

right!